oauth2_provider/forms.py

```diff
@@ -80,7 +80,7 @@ def clean(self):
             except User.DoesNotExist:
                 user = None
 
-        if user is None or not user.is_active:
```

Comment:
I was planning to add a comment with a JIRA ticket number for the pending work.
Note that because of the current coupling of this flag, devOps currently changes the user's password and email address to mark an edX user as "inactive".  @feanil Can you confirm this?